content/sigstore-november-roundup-2023.md

```diff
@@ -0,0 +1,91 @@
++++
+
+title = "Sigstore November Roundup"
+date = "2023-11-30"
+tags = ["sigstore","securesupplychain"]
+draft = false
+author = "Luke Hinds (TSC Chair)"
+type = "post"
+
++++
+
+Welcome to the November edition of the Sigstore Roundup! This is a regular summary of Sigstore news, events, releases and other happenings.
+
+### Sigstore Google Season of Docs 2023 Case Study
+ 
+A very comprehisive case study has been published on the [Sigstore docs wiki](https://github.com/sigstore/docs/wiki/Sigstore-Google-Season-of-Docs-2023-Case-Study) about the Sigstore project's participation in the 2021 program.
+
+Thank you Lisa Tagliaferri for all your hard work on this and making it a success!
+
+### Latest Releases
+
+#### Rekor v1.3.3
+
+Rekor’s aims to provide an immutable tamper-resistant ledger of metadata generated within a software projects supply chain. Its latest release is [v1.3.3](https://github.com/sigstore/rekor/releases/tag/v1.3.3).
+
+What’s changed:
+
+- Add method to get artifact hash for an entry 
```

Comment:
Can we add more items from the changelogs recently? this one is an internal API (not user facing) and reads a bit confusing to me.

Reply:
done!